app/org/sagebionetworks/bridge/json/ScheduleSerializer.java

```diff
@@ -31,14 +34,23 @@ public void serialize(Schedule schedule, JsonGenerator gen, SerializerProvider p
         gen.writeStartObject();
         writeString(gen, LABEL_PROPERTY, schedule.getLabel());
         writeString(gen, CRON_TRIGGER_PROPERTY, schedule.getCronTrigger());
-        if (schedule.getStartsOn() != null && schedule.getStartsOn() != 0L) {
-            writeString(gen, STARTS_ON_PROPERTY, DateUtils.convertToISODateTime(schedule.getStartsOn()));    
+        if (schedule.getStartsOn() != null) {
+            writeString(gen, STARTS_ON_PROPERTY, DateUtils.getISODateTime(schedule.getStartsOn()));    
         }
-        if (schedule.getEndsOn() != null && schedule.getEndsOn() != 0L) {
-            writeString(gen, ENDS_ON_PROPERTY, DateUtils.convertToISODateTime(schedule.getEndsOn()));    
+        if (schedule.getEndsOn() != null) {
+            writeString(gen, ENDS_ON_PROPERTY, DateUtils.getISODateTime(schedule.getEndsOn()));    
         }
-        if (schedule.getExpires() != null && schedule.getExpires() != 0L) {
-            writeString(gen, EXPIRES_PROPERTY, DateUtils.convertToDuration(schedule.getExpires()));    
+        if (schedule.getExpires() != null) {
+            writeString(gen, EXPIRES_PROPERTY, schedule.getExpires().toString());    
```

Comment:
SFAICT, all of the toString() methods in the Joda Time library produce standard ISO 8601 formatted output, e.g. http://joda-time.sourceforge.net/apidocs/org/joda/time/LocalTime.html#toString() where it says as much, or http://joda-time.sourceforge.net/apidocs/org/joda/time/base/AbstractInstant.html#toString(), where it says the same for DateTime.